backend/api/model.py

```diff
@@ -214,7 +214,7 @@ def build_plugin_metadata(plugin: str, version: str) -> Tuple[str, dict]:
     if not metadata:
         return plugin, metadata
     github_repo_url = metadata.get('code_repository')
-    if github_repo_url:
+    if github_repo_url and github_repo_url.startswith("https://github.com/"):
```

Comment:
```suggestion
    if github_repo_url and github_repo_url.startswith(
        ("https://github.com/", "https://www.github.com/", "http://github.com/", "http://www.github.com/")
    ):
```
You might want to check for variants such as these - maybe even allow people to omit the scheme?

Reply:
I am gonna capture this in a backlog issue, as this might require changes in a few other places.